docs/how-to/data/kata-monitor-daemonset.yml

```diff
@@ -26,7 +26,7 @@ spec:
       hostNetwork: true
       containers:
       - name: kata-monitor
-        image: docker.io/katadocker/kata-monitor:2.0.0
+        image: quay.io/kata-containers/kata-monitor:2.0.0
```

Comment:
I wish there was a standard meta-language for yaml. I guess this is fine, but it would be great if we could auto-generate these quay.io references from a `.in` file or similar at some point (with the value coming from `versions.yaml` maybe).

Reply:
Hmm.  That's something that crossed my mind a few times and I couldn't figure out what would be the best approach.
We can have things generated from info we keep in the Makefile, but when we require users to build our project to actually get something they can use ... or we keep things as they are and users can consume those directly from our repo.  And I still am not decided on what's the best approach.